lib/watir/elements/element.rb

```diff
@@ -493,6 +494,21 @@ def browser
       @query_scope.browser
     end
 
+    #
+    # Returns true if a previously located element is no longer attached to DOM.
+    #
+    # @return [Boolean]
+    # @see Watir::Wait
+    #
+
+    def stale?
+      raise Watir::Exception::Error, "Can not check staleness of unused element" unless @element
```

Comment:
This method behavior is definitely a bit different from others. I now understand why it is, so I guess we can keep it as is and see if users complain.